sdk/iothub/src/service/mod.rs

```diff
@@ -268,10 +273,155 @@ impl ServiceClient {
             response_time_out,
         )
     }
+
+    /// Get the module twin of a given device and module
+    ///
+    /// ```
+    /// use iothub::service::ServiceClient;
+    ///
+    /// # let connection_string = "HostName=cool-iot-hub.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=YSB2ZXJ5IHNlY3VyZSBrZXkgaXMgaW1wb3J0YW50Cg==";
+    /// let iothub = ServiceClient::from_connection_string(connection_string, 3600).expect("Failed to create the ServiceClient!");
+    /// let twin = iothub.get_module_twin("some-device", "some-module");
+    /// ```
+    pub async fn get_module_twin<S, T>(
+        &self,
+        device_id: S,
+        module_id: T,
+    ) -> Result<ModuleTwin, AzureError>
+    where
+        S: Into<String>,
+        T: Into<String>,
+    {
+        get_module_twin(&self, device_id.into(), module_id.into()).await
```

Comment:
What's the reason for this wrapper function?

Reply:
I did it this way mostly to keep the logic (actual rest call) close to the rest of the twin related functionality (e.g. structs/enums/etc) and keeping the interface as clean as possible. I get that this may seem a little redundant though so I can definitely move that over to here if that's desirable!

Reply:
In hindsight, implementation is kept in `twin.rs` anyway because of the `get_twin` helper function. I removed the redundant functions, thanks!